client-side/src/Components/studentComponent/Gatepasses/LocalFlexible/Form/LFform.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState, useReducer } from "react";
+import ReactDropdown from "react-dropdown";
+import moment from 'moment';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import "./style.scss";
+
+
+const LFform = (props) => {
+  const[hour,settime]=useState(new Date().getHours());// present time
```

Comment:
setState should always be camelcase, and the name in the setState should be that of the state you have made. For ex - 
[user, setUser] is correct, but not [user,setCandidate]

Reply:
corrected